packages/vue/src/components/__tests__/alias-control.spec.ts

```diff
@@ -0,0 +1,79 @@
+import AliasControl from '../alias-control.vue';
+import { components } from '../../../global-spec';
+import { render, screen } from '@testing-library/vue';
+
+// mock Math.random so that ids are statically generated
+jest.spyOn(global.Math, 'random').mockReturnValue(0.123456789);
+
+describe('AliasControl Component', () => {
+  describe('Label', () => {
```

Comment:
Thinking this `describe` (and the `describe` on ln 43) are extraneous?

Reply:
Removed the extra layer in [0d50811](https://github.com/aws-amplify/amplify-ui/pull/2712/commits/0d50811c7c1ba607eee7e01d5deb70f39c99a7ac)